src/ice_shelf/MOM_ice_shelf_dynamics.F90

```diff
@@ -276,7 +276,7 @@ subroutine register_ice_shelf_dyn_restarts(G, US, param_file, CS, restart_CS)
     allocate( CS%ground_frac(isd:ied,jsd:jed), source=0.0 )
     allocate( CS%taudx_shelf(IsdB:IedB,JsdB:JedB), source=0.0 )
     allocate( CS%taudy_shelf(IsdB:IedB,JsdB:JedB), source=0.0 )
-    allocate( CS%bed_elev(isd:ied,jsd:jed) ) ; CS%bed_elev(:,:) = G%bathyT(:,:) + G%Z_ref
+    allocate( CS%bed_elev(isd:ied,jsd:jed), source=0.0 )
```

Comment:
Space following `( ` and preceding ` )` should be removed, but adjacent lines have the same issue so not necessary at this time.